edx_exams/apps/core/management/commands/test/test_bulk_add_course_staff.py

```diff
@@ -0,0 +1,114 @@
+""" Tests for bulk_add_course_staff management command """
+from tempfile import NamedTemporaryFile
+
+from django.core.management import call_command
+from django.test import TestCase
+
+from edx_exams.apps.core.models import CourseStaffRole, User
+from edx_exams.apps.core.test_utils.factories import UserFactory
+
+
+class TestBulkAddCourseStaff(TestCase):
+    """ Test bulk_add_course_staff management command """
+
+    def setUp(self):
+        super().setUp()
+        self.command = 'bulk_add_course_staff'
+        self.success_log_message = 'Bulk add course staff complete!'
+
+        # create existing user
+        self.user = UserFactory.create(
+            username='amy',
+            is_active=True,
+            is_staff=True,
+        )
+        self.user.email = 'amy@pond.com'
+        self.user.save()
+
+        self.course_id = 'course-v1:edx+test+f19'
+        self.course_role = 'staff'
+
+    def _write_test_csv(self, csv, lines):
+        """ Write a test csv file with the lines provided """
+        csv.write(b'username,email,role,course_id\n')
+        for line in lines:
+            csv.write(line.encode())
+        csv.seek(0)
+        return csv
+
+    def _assert_user_and_role(self, username, email, course_role, course_id):
+        """ Helper that asserts that User and CourseStaffRole are created """
+        user = User.objects.filter(username=username, email=email)
+        assert user.exists()
+        assert CourseStaffRole.objects.filter(
+            user=user[0].id,
+            course_id=course_id,
+            role=course_role,
+        ).exists()
+
+    def test_empty_csv(self):
+        lines = []
+        with NamedTemporaryFile() as csv:
+            csv = self._write_test_csv(csv, lines)
+            call_command(self.command, f'--csv_path={csv.name}')
```

Comment:
Should we assert something here, like no `CourseStaffRole` was created for `self.user`?

Reply:
Good call, just added that!